aws-creds/iam.tf

```diff
@@ -137,12 +201,37 @@ data "aws_iam_policy_document" "terraform_iam_policy_source" {
       "ec2:GetLaunchTemplateData",
       "ec2:ModifyLaunchTemplate",
       "ec2:RunInstances",
+      "ecr:CreateRepository",
+      "efs:CreateFileSystem",
       "eks:CreateCluster",
       "eks:DeleteCluster",
       "eks:DescribeCluster",
       "eks:ListClusters",
       "eks:UpdateClusterConfig",
       "eks:DescribeUpdate",
+      "iam:GetInstanceProfile",
+		  "iam:GetOpenIDConnectProvider",
```

Comment:
use spaces to match the other lines?

Reply:
spaces instead of tabs everywhere i can is done